### ISSUE TITLE
Cleanup formatter API

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brailleapps/dotify#70]
```
The formatter API contains actual implementations, as well as previous 
interfaces that are no longer relevant from an API standpoint.

```

Original issue reported on code.google.com by `joel.hak...@mtm.se` on 8 Nov 2013 at 9:18
- Blocked on: #73
